stdlib/public/Platform/Platform.swift

```diff
@@ -94,36 +94,18 @@ public func || <T : Boolean>(
 // sys/errno.h
 //===----------------------------------------------------------------------===//
 
+@_silgen_name("_swift_Platform_getErrno")
+func getErrno() -> Int32
```

Comment:
For consistency with other shims, could you use the full name, `func _swift_Platform_getErrno `?  I don't like that the name is short and nice, when it is not the primary entry point, overlay developers should not be tempted to call it.